modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -308,6 +322,51 @@ private ObjectField _addObjectField(
 		return objectFieldPersistence.update(objectField);
 	}
 
+	private void _addObjectFieldSettings(
+			long userId, long objectFieldId,
+			List<ObjectFieldSetting> newObjectFieldSettings)
+		throws PortalException {
+
+		List<ObjectFieldSetting> oldObjectFieldSettings =
+			_objectFieldSettingPersistence.findByObjectFieldId(objectFieldId);
+
+		for (ObjectFieldSetting oldObjectFieldSetting :
+				oldObjectFieldSettings) {
+
+			Stream<ObjectFieldSetting> stream = newObjectFieldSettings.stream();
+
+			Optional<ObjectFieldSetting> optional = stream.filter(
+				newObjectFieldSetting ->
+					newObjectFieldSetting.getName() ==
```

Comment:
Is there any particular reason that led you to choose the `==` operator to compare strings?

Reply:
No reason, it was a copy and paste actually. I'll change it to `Objects.equals` 😄 
Thanks!